{"additions": 1, "auther_ref": "fixmuonCut", "auther_sha": "c3c74a2d6ca9b4ef161ea5e20eb52905f2acf942", "author": "boudoul", "body": "as mentioned in this Issue : #26771 (thanks @christopheralanwest , and also @abdoulline  ,  @mazarkin for noticing ) there was a mistake introdcued after the 2019-->2021 migration , a fileptah was incorrect - This PR is fixing this issue. \r\n\r\n#### PR validation:\r\nIt should not affect any std WF (where the geometry is read via the DB) , but I ran few events calling explicitely the 2021 XML : \r\nrunTheMatrix.py -l 11623 --command '--geometry Extended2021'\r\n\r\n", "branch": "master", "changed_files": 1, "comments": 5, "commits": 1, "created_at": "1558442725", "deletions": 1, "labels": ["bug-fix", "code-checks-approved", "comparison-pending", "geometry-rejected", "orp-pending", "pending-signatures", "tests-pending", "upgrade-pending"], "milestone": "CMSSW_11_0_X", "number": 26856, "release-notes": [], "review_comments": 4, "state": "open", "title": "Fix file path forgotten after 2019-->2021 migration (XML geometry) ", "updated_at": "1558444267", "user": "boudoul"}